packages/ui/src/components/Tabs/tabs.tsx

```diff
@@ -16,6 +17,10 @@ export type TabItem = {
     children?: React.ReactNode
 }
 
+type CondoTabsTypeProp = {
+    type?: 'rounded'
```

Comment:
Why not just `rounded?: bool`. Type is fine, but where is default type then?

Reply:
We didnt take type prop from default ant props before
For my task I didnt need it so decided that for now I would create my own type prop. And if we need to add default type in the future it will be easier to match default types with rounded type than to match how rounded property should behave with type prop

Reply:
Didn't get it. There's 2 options to do it.
1. Modifier. This way you can pass `rounded` modifier to turn behaviour on or not pass. However, you can specify it directly like: `rounded={true/false}`
2. Select. This way you can specify option by `myAtrribute='value'` prop, or not specify to have default value. However you should be able to specify it directly as in modifier above: `myAttrubute='defaultOption'`.

In both cases you use explicit default value if you want to control appearance based on state (programmatically):
```typescript
const tabsType = condition ? 'rounded' : 'default'
``` 
Right now to do it with type approach, I need to write something like:
```typescript
const tabsType = condition ? 'rounded' : undefined
``` 
Which is way worse than having a default constant or using modifier

Reply:
Another reason in favor of the modifier is the consistency of the API. 

Let's imagine that tomorrow we want a rounded button (or input or something else). Would it be a type? I don't think so. Because any button or input can be rounded.

Then another question follows: Why do we have rounded as a modifier in one place and a type in another? There is an inconsistency in the API, making it more difficult to use the API.